app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java

```diff
@@ -0,0 +1,112 @@
+import android.os.Bundle;
+import android.view.ContextThemeWrapper;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.Nullable;
+import androidx.databinding.DataBindingUtil;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.eventyay.organizer.BuildConfig;
+import com.eventyay.organizer.R;
+import com.eventyay.organizer.common.mvp.view.BaseFragment;
+import com.eventyay.organizer.core.settings.AcknowledgementDecider;
+import com.eventyay.organizer.core.settings.LegalPreferenceFragment;
+import com.eventyay.organizer.databinding.AboutUsFragmentBinding;
+import com.eventyay.organizer.utils.BrowserUtils;
+
+import mehdi.sakout.aboutpage.AboutPage;
+import mehdi.sakout.aboutpage.Element;
+
+public class AboutUsFragment extends BaseFragment {
+
+    private AboutUsFragmentBinding binding;
+    private final AcknowledgementDecider acknowledgementDecider = new AcknowledgementDecider();
+
+    public static AboutUsFragment newInstance() {
+        return new AboutUsFragment();
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        final Context contextThemeWrapper = new ContextThemeWrapper(getActivity(), R.style.AppTheme);
+        LayoutInflater localInflater = inflater.cloneInContext(contextThemeWrapper);
+        binding =  DataBindingUtil.inflate(localInflater, R.layout.about_us_fragment, container, false);
+
+        Element legalElement = new Element();
+        legalElement.setTitle("Legal");
+
+        legalElement.setOnClickListener(v -> {
+            FragmentTransaction transaction = getFragmentManager().beginTransaction();
+            transaction
+                .replace(R.id.fragment, LegalPreferenceFragment.newInstance())
+                .addToBackStack(null)
+                .commit();
+        });
+
+        Element developersElement = new Element();
+        developersElement.setTitle(getString(R.string.developers));
+
+        developersElement.setOnClickListener(v -> {
+            BrowserUtils.launchUrl(getContext(), getString(R.string.github_developers_link));
+        });
+
+        Element shareElement = new Element();
+        shareElement.setTitle(getString(R.string.share));
+
+        shareElement.setOnClickListener(v -> {
+            Intent sharingIntent = new Intent();
+            sharingIntent.setAction(Intent.ACTION_SEND);
+            sharingIntent.setType("text/plain");
+            String shareSub = getString(R.string.msg_check_out);
+            String shareBody = getString(R.string.msg_check_this_out) + getString(R.string.play_store_link);
+            String shareTitle = getString(R.string.msg_share_using);
+            sharingIntent.putExtra(Intent.EXTRA_SUBJECT, shareSub);
+            sharingIntent.putExtra(Intent.EXTRA_TEXT, shareBody);
+            startActivity(Intent.createChooser(sharingIntent, shareTitle));
```

Comment:
Duplicate sharing code

Reply:
How?

Reply:
I've removed it from the Device Settings.

Reply:
And the sharing code in Utils is for sharing an event.

Reply:
OK

Reply:
> Everything is hardcoded. It should be configurable

Configurable from where?